package-scripts.js

```diff
@@ -39,20 +38,22 @@ module.exports = {
       stories: "eslint --color stories",
       storybook: "eslint --color --no-ignore .storybook/config.js",
       test: "eslint --color test",
+      ts: npsUtils.series.nps("build-package-libs", "compile-ts"),
       default: npsUtils.series.nps(
         "lint.test",
         "lint.stories",
         "lint.storybook",
         "lint.demo",
-        "lint.src"
+        "lint.src",
+        "lint.ts"
       )
     },
     format: {
       default: 'prettier --write "./**/*.{js,jsx,json,ts,tsx}"',
       ci: 'prettier --list-different "./**/*.{js,jsx,json,ts,tsx}"'
     },
     check: {
-      ci: npsUtils.series.nps("format.ci", "lint", "test.ci"),
+      ci: npsUtils.series.nps("format.ci", "lint", "karma.ci"),
```

Comment:
The lint process for typescript involves running `build-package-libs`, which `test.ci` also did, so I removed `test.ci` and replaced it with `karma.ci` since the packages will already be built for karma.  

Let me know if this doesn't make sense or might produce some other problems, I just didn't want `build-package-libs` to run twice during ci if we didn't need it.